demo/m1_facilitator/bin/request_deposit_testnet.ts

```diff
@@ -0,0 +1,117 @@
+import fs from 'fs';
+import path from 'path';
+
+import Deposit from '../lib/Deposit';
+
+const inquirer = require('inquirer');
+const Web3 = require('web3');
+
+const { BN } = new Web3().utils;
+
+interface DepositInputInfo {
+  web3EndPoint: string;
+  valueTokenAddress: string;
+  erc20GatewayAddress: string;
+  transactionGasPrice: string;
+  depositorKeystore: string;
+  password: string;
+  amountToDeposit: string;
+  beneficiary: string;
+  gasPrice: string;
+  gasLimit: string;
+}
+
+async function readInput(): Promise<DepositInputInfo> {
+  const filePath = path.join(__dirname, '/', 'depositor.json');
+  const answer = await inquirer.prompt([
+    {
+      type: 'string',
+      name: 'web3EndPoint',
+      default: 'https://rpc.slock.it/goerli',
+      message: 'Enter origin chain(goerli) end point:',
+    },
+    {
+      type: 'string',
+      name: 'valueTokenAddress',
+      default: '0xd426b22f3960d01189a3d548b45a7202489ff4de',
+      message: 'Enter value token address:',
+    },
+    {
+      type: 'string',
+      name: 'erc20GatewayAddress',
+      default: '0x9B0fd9FB015d9311738ed5aECfF3A626e7A149C1',
+      message: 'Enter ERC20 gateway address:',
+    },
+    {
+      type: 'string',
+      name: 'transactionGasPrice',
+      default: '0x3B9ACA00',
+      message: 'Enter transaction gas price:',
+    },
+    {
+      type: 'string',
+      name: 'depositorKeystore',
+      message: 'Enter depositor keystore filepath:',
+      default: filePath,
+      validate(input: string): boolean {
+        return fs.existsSync(input);
+      },
+    },
+    {
+      type: 'password',
+      name: 'password',
+      message: 'Enter depositor keystore password:',
+    },
+    {
+      type: 'string',
+      name: 'amountToDeposit',
+      message: 'Enter amount to deposit in (wei):',
+      validate(input: string): number {
+        return new BN(input).gtn(0);
+      },
+    },
+    {
+      type: 'string',
+      name: 'beneficiary',
+      message: 'Enter beneficiary address on the metachain(Hadapsar-1405):',
+      validate(input: string): string {
```

Comment:
Add `depositor address` as a beneficiary address as a default.

Reply:
The path to the Keystore is taken using the inquirer. So, till this inquirer prompt is not completed we won't get the address(using the Keystore). 
I think we can keep it this way only.

Reply:
Handled it using another inquirer to get the default value of the beneficiary address.